### PR TITLE
Update command-line-parameter-examples.md

### PR DESCRIPTION
**Pull request by frayus**
Adds more information on using the --wait parameter.  Some command utilities require additional parameters to have the installer wait and to get its return value.

(You can replace all of this text with your description.)

Before creating your pull request, please check your content against these quality criteria:

- Did you consider search engine optimization (SEO) when you chose the title in the metadata section and the H1 heading (i.e. the displayed title that starts with a single #)?
- For new articles, did you add it to the table of contents?
- Did you update the "ms.date" metadata for new or significantly updated articles?
- Are technical terms and concepts introduced and explained, and are acronyms spelled out on first mention?
- Should this page be linked to from other pages or Microsoft web sites?

For more information about creating content for docs.microsoft.com, see the contributor guide at https://docs.microsoft.com/contribute/.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@frayus : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

File: docs/install/command-line-parameter-examples.md [hunk 1]
```diff
@@ -54,9 +54,16 @@ For lists of the workloads and components that you can install by using the comm
    --includeRecommended --quiet --wait
   ```
 
-  > [!NOTE]
-  > The `--wait` parameter is designed for use in a batch file. In a batch file, execution of the next command will not continue until the installation has completed. The `%ERRORLEVEL%` environment variable will contain the return value of the command, as documented in the [Use command-line parameters to install Visual Studio](use-command-line-parameters-to-install-visual-studio.md) page.
+## Using --wait
+
+* Use in batch files or scripts to wait for the Visual Studio installer to complete before the next command is executed. For batch files, an`%ERRORLEVEL%` environment variable will contain the return value of the command, as documented in the [Use command-line parameters to install Visual Studio](use-command-line-parameters-to-install-visual-studio.md) page. Some command utilities require additional parameters to wait for completion and to get the installer's return value. The following is an example of the additional parameters used with the powershell script command 'Start-Process':
```
Comment (TerryGLee) [inline docs/install/command-line-parameter-examples.md]: Please rewrite "powershell" as "PowerShell". For more information, see the [PowerShell documentation](https://docs.microsoft.com/en-us/powershell/) page.

Comment (TerryGLee) [reply docs/install/command-line-parameter-examples.md]: Review changes, per the above comment.

**Comment by TerryGLee** [comment]
#sign-off

**Pull request merged**